packages/expo-updates/src/UseUpdates.types.ts

```diff
@@ -0,0 +1,146 @@
+   * UUID is represented in its canonical string form (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`) and
+   * will always use lowercase letters. In development mode, or any other environment in which
+   * `expo-updates` is disabled, this value is undefined.
+   */
+  updateId?: string;
+  /**
+   * The channel name of the current build, if configured for use with EAS Update; undefined otherwise.
+   */
+  channel?: string;
+  /**
+   * If `expo-updates` is enabled, this is a `Date` object representing the creation time of the update
+   * that's currently running (whether it was embedded or downloaded at runtime).
+   *
+   * In development mode, or any other environment in which `expo-updates` is disabled, this value is
+   * undefined.
+   */
+  createdAt?: Date;
+  /**
+   * This will be true if the currently running update is the one embedded in the build,
+   * and not one downloaded from the updates server.
+   */
+  isEmbeddedLaunch: boolean;
+  /**
+   * `expo-updates` does its very best to always launch monotonically newer versions of your app so
+   * you don't need to worry about backwards compatibility when you put out an update. In very rare
+   * cases, it's possible that `expo-updates` may need to fall back to the update that's embedded in
+   * the app binary, even after newer updates have been downloaded and run (an "emergency launch").
+   * This boolean will be `true` if the app is launching under this fallback mechanism and `false`
+   * otherwise. If you are concerned about backwards compatibility of future updates to your app, you
+   * can use this constant to provide special behavior for this rare case.
+   */
+  isEmergencyLaunch: boolean;
+  /**
+   * If `expo-updates` is enabled, this is the
+   * [manifest](https://docs.expo.dev/versions/latest/sdk/updates/#updatesmanifest) object for the update that's currently
+   * running.
+   *
+   * In development mode, or any other environment in which `expo-updates` is disabled, this object is
+   * empty.
+   */
+  manifest?: Partial<Manifest>;
+  /**
+   * The runtime version of the current build.
+   */
+  runtimeVersion?: string;
+};
+
+/**
+ * The different possible types of updates.
+ * Currently, the only supported type is `UpdateInfoType.NEW`.
+ */
+export enum UpdateInfoType {
+  /**
+   * This is the type for new updates found on or downloaded from the update server.
+   */
+  NEW = 'new',
+}
+
+/**
+ * Structure representing an available or downloaded update.
+ */
+export type UpdateInfo = {
+  /**
+   * The type of update.
+   */
+  type: UpdateInfoType;
+  /**
+   * For updates of type `UpdateInfoType.NEW`, this is
+   * a string that uniquely identifies the update. For the manifests used in the current Expo Updates protocol (including
+   * EAS Update), this represents the update's UUID in its canonical string form (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`)
+   * and will always use lowercase letters.
+   */
+  updateId?: string;
```

Comment:
edit: it appears we can't do this until we remove classic updates stuff.

```suggestion
  updateId: string;
```